src/hotspot/share/opto/idealGraphPrinter.cpp

```diff
@@ -448,6 +448,11 @@ void IdealGraphPrinter::visit_node(Node* n, bool edges) {
         }
       }
     }
+    if (n->adr_type() != nullptr) {
+      stringStream adr_type_stream;
```

Comment:
Other stringStream around are using a preallocated buffer. Would it be a good idea here too?

Reply:
I think that makes sense. Thanks.

No strong opinion whether we should change what is already there, as long as we don't add more.